src/lao.c

```diff
@@ -323,6 +353,14 @@ static int l_is_big_endian(lua_State *L)
 	return 1;
 }
 
+static int l_append_global_option(lua_State *L)
+{
+	const char *key = luaL_checkstring(L, 1);
+	const char *val = luaL_checkstring(L, 2);
```

Comment:
I was going to suggest that a `val` of `true` would map to NULL for use options like `"quiet"` which have optional values (see https://xiph.org/ao/doc/config.html)
libao converts `NULL` to the empty string https://github.com/xiph/libao/blob/master/src/audio_out.c#L1300 so it's safe